ideas/100-smooth-chat-ux.md

```diff
@@ -0,0 +1,119 @@
+<!-- Please Review https://docs.google.com/document/d/1CaFM2ZXGOKf05_LXMPJeNNy5qJOdAq91EF2Gn2QUBFI/edit# for more details -->
+<!-- in PR the document should be named as`DEV#1-title.md` -->
+
+## Preamble
+
+    Idea: UX 100
+    Title: Smooth chat UX
+    Status: Draft
+    Created: date created on 2018-03-23
+
+
+## Summary
+<!-- "If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the Idea. -->
+
+The main goal of this UX swarm is to "smooth the edges" of our chat experience for our upcoming launch. Our current chat implementation works, but it is still not optimal. Two of the core OKR's (Use Status every day and message is reliable) depends heavily on a good and smooth chat experience.
+
+The UX issues are split between 3 groups, starting with the low effort tasks/high payoff moving to big effort/high payoff (v1, v2, v3). Although its scope can seem ambitious at first, the idea is to have deliverables once in two weeks. Those deliverables will contain all the necessary documentation necessary for implementation. More designers are welcome to join as we go, as well as developers, QA's PM's.
+
+## Swarm Participants
+<!-- Each contributor pledges to the idea with their FOCUS value. (hours per week) -->
+<!-- Here all roles in swarm are defined and filled, one of the contributors should responsibility of the Idea as Lead. -->
+
+<!-- Testing/Evaluation support role is also mandatory to check in on specified Goal dates or earlier. -->
+
+<!-- Lead Contributor is the Owner of the Idea. If required, they can get support from a PM, but should be responsible for end to end execution of the Idea. This includes ensuring appropriate resources are allocated, setting realistic timelines and milestones, and any post-launch metrics or bug fixes that are attributed to the Idea -->
+<!-- A swarm requires at minimum 3 contributors and 1 evaluator/tester -->
+<!-- 'Contributor' should be replaced with a descriptive role type. -->
+- Lead Contributor: @ricardosaavedra
+- Testing & Evaluation: <!-- @username -->
```

Comment:
What does the interest look like from other core contributors (Clojure devs, UX(R), PM) so far?